lego/settings/lego.py

```diff
@@ -57,4 +57,4 @@
 GSUITE_GROUPS = []
 
 #  External users in GSuite not managed by lego. (Don't suspend these.)
-GSUITE_EXTERNAL_USERS = ["admin@abakus.no", "pr-deprecated@abakus.no"]
+GSUITE_EXTERNAL_USERS = ["admin@abakus.no"]
```

Comment:
Why remove this?

Reply:
Hmmm. I changed this in https://github.com/webkom/lego/commit/344d3ef4ff5aee8ddd4abf7be624e2170a48ed8c, and it is merged into master..